### ISSUE TITLE
python-trezor: missing attrs from requirements.txt/setup.py

### ISSUE THREAD

**Issue by prusnak** [trezor/trezor-firmware#1232]
running `trezorctl` from 0.12.2 release:

```
ModuleNotFoundError: No module named 'attr'
```

can we add `attrs` to `requirements.txt`/`setup.py`?

Even better - can we have some check for this so we never miss adding a new dependency in the future?

---

## WORKAROUND

install `attrs` the same way you installed `trezor`.

e.g., in your terminal:
```sh 
pip3 install attrs
```

**Comment by prusnak** [comment]
It seems there is a flake plugin for this: https://pypi.org/project/flake8-requirements/

**Comment by matejcik** [comment]
this particular thing should have been found by `tox`, i'll need to investigate why that didn't happen

**Comment by matejcik** [comment]
ok, this is fun: `attrs` is a dependency of several things in the dev environment, and tox is testing inside that.

**Comment by pradysp** [comment]
Hello is there a workaround for this bug ? I cant even run the basic '**trezorctl**' command now.

**Comment by prusnak** [comment]
@pradysp which distro are you using?

**Comment by pradysp** [comment]
@prusnak Using Anaconda distribution of python version 3.6, trezor 0.12.2 release

**Comment by prusnak** [comment]
@pradysp install the `attrs` package into your environment

**Comment by pradysp** [comment]
@

> @pradysp install the `attrs` package into your environment: most probably `pip install attrs`

That fixed it. Thanks

**Comment by matejcik** [comment]
fixed and added a check in 6dda240f

**Comment by AdamSHurwitz** [comment]
The workaround with `pip3 install attrs` is still required in *version 0.12.2* or else an error is thrown that `attrs` is not found.
